metadata/2024-10-25-signer-native-format-fix/enable_signer_native_format_fix.json

```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Enable the signer native format fix",
+  "description": "Fixes the issue of string_utils::native_format in formatting signer values",
+  "source_code_url": "https://github.com/aptos-labs/aptos-core",
+  "discussion_url": "https://github.com/aptos-labs/aptos-core"
```

Comment:
could you please update this to the issue url where you filed the bug?

Reply:
Updated the URL.